src/WorkOS.net/Services/UserManagement/UserManagementService.cs

```diff
@@ -140,6 +140,25 @@ public UserManagementService(WorkOSClient client)
             return await this.Client.MakeAPIRequest<(User, Session)>(request, cancellationToken);
         }
 
+        /// <summary>
+        /// Creates a one-time Magic Auth code and emails it to the user.
+        /// </summary>
+        /// <param name="options"> Parameters used to send magic auth code.</param>
+        /// <param name="cancellationToken">An optional token to cancel the request.</param>
+        /// <returns> A Magic Auth Challenge.</returns>
+        public async Task<User> SendMagicAuthCode(
+            SendMagicAuthCodeOptions options,
+            CancellationToken cancellationToken = default)
+        {
+            var request = new WorkOSRequest
+            {
+                Options = options,
+                Method = HttpMethod.Post,
+                Path = $"/users/magic_auth/send",
+            };
+            return await this.Client.MakeAPIRequest<User>(request, cancellationToken);
```

Comment:
We'll need to add a separate response DTO here, as the user is nested under the `user` key:

```json
{
  "user": {
    "object": "user",
    "id": "user_",
    // ...
  }
}

```

Reply:
Not sure where I see user nested under the user key based on the doc 
![CleanShot 2023-08-28 at 15 36 55@2x](https://github.com/workos/workos-dotnet/assets/5375567/cdc09a1b-ef74-4c7b-b0b6-c9954d08385b)
![CleanShot 2023-08-28 at 15 36 38@2x](https://github.com/workos/workos-dotnet/assets/5375567/6a02e7c0-d355-4106-b2e1-95ad89684452)

Reply:
It's the "Returns `object`" with the `user` nested underneath:

<img width="1197" alt="Screenshot 2023-08-28 at 3 56 06 PM" src="https://github.com/workos/workos-dotnet/assets/1486634/eaffb9c6-dc9a-42f4-be98-6bec2a97b182">

Reply:
Ah got it, is there a reason for returning just an object with only a user nested instead of just returning a user object?

Reply:
Added DTO `SendMagicAuthCodeResponse`